src/ol/Image.js

```diff
@@ -222,4 +257,67 @@ export function listenImage(image, loadHandler, errorHandler) {
   };
 }
 
+/**
+ * Loads an image.
+ * @param {HTMLImageElement} image Image, not yet loaded.
+ * @param {string} [src] `src` attribute of the image. Optional, not required if already present.
+ * @return {Promise<HTMLImageElement>} Promise resolving to an `HTMLImageElement`.
+ * @api
+ */
+export function load(image, src) {
+  if (src) {
+    image.src = src;
+  }
```

Comment:
Maybe this doesn't make any difference given current implementations, but it feels like it would be safer to assign the `image.src` after registering the listeners below.